code/modules/paperwork/photography.dm

```diff
@@ -386,9 +390,14 @@
 	user << "<span class='notice'>[pictures_left] photos left.</span>"
 	icon_state = "camera_off"
 	on = 0
-	spawn(64)
-		icon_state = "camera"
-		on = 1
+	if(!remote)
```

Comment:
could just do var/time = remote ? 600 : 64    i think this syntax could work, not totally sure tho